packages/components/src/dateInput/DateInput.js

```diff
@@ -13,7 +13,8 @@ import {
 
 export type Props = {|
   +label?: React.Node,
-  +labelContainerStyle?: StylePropType,
+  +containerStyle?: StylePropType,
```

Comment:
Why not to use the same convention also here?

containerStyle => formLabelContainerStyle
labelStyle => formLabelTextStyle

Reply:
@jvaclavik Done! I think the PR is ready now 👍